CHANGELOG.md

```diff
@@ -1,3 +1,124 @@
+
+* Improves performance of blocking shard moves
+
+* Uses a faster custom copy logic for non-blocking shard moves
+
+* `isolate_tenant_to_new_shard()` adds support for columnar tables
+
+* `isolate_tenant_to_new_shard()` adds support for partitioned tables
+
+* `isolate_tenant_to_new_shard()` drops support for replicated tables
+
+* `isolate_tenant_to_new_shard()` now fails when run concurrently with itself
+
+* Adds an `allow_unsafe_constraints` flag for constraints without distribution
+  column
+
+* Adds support for `GRANT/REVOKE` on aggregates
+
+* Adds support for `NULLS NOT DISTINCT` clauses for indexes
+
+* Adds support for setting relation options for columnar tables using
+  `ALTER TABLE`
+
+* Adds support for unlogged distributed sequences
+
+* Checks existence of the shards before insert, delete, and update
+
+* Creates all foreign keys quickly at the end of a shard move
+
+* Hides tables owned by extensions from `citus_tables` and `citus_shards`
+
+* Introduces GUC `citus.skip_constraint_validation`
+
+* Introduces `citus_locks` view
+
+* Propagates `VACUUM` and `ANALYZE` to worker nodes
+
+* Makes non-partitioned table size calculation quicker
+
+* Only shows shards in moving state in `get_rebalance_progress()`
+
+* Removes `do_repair` option from `citus_copy_shard_placement()`
+
+* Removes deprecated re-partitioning functions like
+  `worker_hash_partition_table()`
+
+* Replaces `citus.hide_shards_from_app_name_prefixes` GUC with
+  `citus.show_shards_for_app_name_prefixes`
+
+* Shows `citus_copy_shard_placement()` progress in `get_rebalance_progres()`
+
+* Shows local tables added to metadata in `citus_tables`
+
+* Supports changing CPU priorities for backends and shard moves
+
+* Tightens security and improves visibility for columnar table access method
+
+* `citus_move_shard_placement()` becomes a noop if shard already exists on node
+
+* `create_distributed_table()` creates new colocation entries when using
+  `colocate_with => 'none'`
+
+* Makes sure that `SELECT . FOR UPDATE `opens a transaction block when used in
+  a function call
+
+* Makes sure to disallow usage of SQL functions referencing to a distributed
+  table and prevents a segfault
+
+* Prevents creating a new colocation entry when replicating reference tables
+
+* Fixes a bug in query escaping in `undistribute_table()` and
+  `alter_distributed_table()`
+
+* Fixes a bug preventing the usage of `isolate_tenant_to_new_shard()` with text
+  column
+
+* Fixes a bug that caused `GRANT` to propagate within `CREATE EXTENSION`
+
+* Fixes a bug that causes incorrectly marking `metadatasynced` flag for
+  coordinator
+
+* Fixes a bug that may cause Citus not to create function in transaction block
+  properly
+
+* Fixes a bug that prevents promoting read-replicas as primaries
+
+* Fixes a bug that prevents setting colocation group of a partitioned
+  distributed table to `none`
+
+* Fixes a bug that prevents using `auto` option for `VACUUM (INDEX_CLEANUP)`
```

Comment:
```suggestion
* Fixes a bug that prevents using `AUTO` option for `VACUUM (INDEX_CLEANUP)`
```

This is part of the SQL syntax. I guess it is better to have it in caps as the PG docs does it.